components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -5139,6 +5140,7 @@ private int addAuthenticator(Connection conn, int tenantId, String idpName, Stri
             prepStmt.setString(5, "1");
             prepStmt.setString(6, authenticatorDispalyName);
             prepStmt.setString(7, definedByType);
+            prepStmt.setString(8, AuthenticationType.IDENTIFICATION.toString());
```

Comment:
Can we say by default unless specified authentication type is identified as of 'IDENTIFICATION' ?

Reply:
Addressed with commit c4009b8301f34d84dcfd97dfe775da67c9fd14d8